react-stopwatch-component/src/stopwatch.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+let timerId = null;
+class Stopwatch extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleClick = this.handleClick.bind(this);
+    this.handleReset = this.handleReset.bind(this);
+    this.state = {
+      isTicking: false,
+      counter: 0
+    };
+  }
+
+  handleClick() {
+    if (!this.state.isTicking) {
+      this.setState({
+        isTicking: true,
+        counter: this.state.counter
+      });
+      timerId = setInterval(() => {
```

Comment:
instead of storing `timerId` in a global variable, assign it to `this`

```js
this.timerId = setInterval(() => {
```

That way, you can put multiple timers on the page without their `timerId`s conflicting.